src/storage/object/list.js

```diff
@@ -9,38 +9,18 @@
  * OF ANY KIND, either express or implied. See the License for the specific language
  * governing permissions and limitations under the License.
  */
-import {
-  S3Client,
-  ListObjectsV2Command,
-} from '@aws-sdk/client-s3';
 
-import getS3Config from '../utils/config.js';
 import formatList from '../utils/list.js';
 
-function buildInput({ org, key }) {
-  return {
-    Bucket: `${org}-content`,
-    Prefix: key ? `${key}/` : null,
-    Delimiter: '/',
-  };
-}
-
 export default async function listObjects(env, daCtx) {
-  const config = getS3Config(env);
-  const client = new S3Client(config);
-
-  const input = buildInput(daCtx);
-  const command = new ListObjectsV2Command(input);
+  const input = {
+    prefix: daCtx.key ? `${daCtx.org}/${daCtx.key}/` : `${daCtx.org}/`,
+    delimiter: '/',
+  };
   try {
-    const resp = await client.send(command);
-    // console.log(resp);
-    const body = formatList(resp, daCtx);
-    return {
-      body: JSON.stringify(body),
-      status: resp.$metadata.httpStatusCode,
-      contentType: resp.ContentType,
-    };
+    const resp = await env.DA_CONTENT.list(input);
```

Comment:
I see that DA_CONTENT is defined in miniflare, but I guess we also need it in the wrangler.toml

Reply:
Yeah, i haven't added it yet. Will do that on one of these PRs